src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -450,6 +450,66 @@ public override async Task<HttpResponseMessage> InvokeMethodWithResponseAsync(Ht
             }
         }
 
+        /// <summary>
+        /// <para>
+        /// Creates an <see cref="HttpClient"/> that can be used to perform Dapr service invocation using <see cref="HttpRequestMessage"/>
+        /// objects.
+        /// </para>
+        /// <para>
+        /// The client will read the <see cref="HttpRequestMessage.RequestUri" /> property, and 
+        /// interpret the hostname as the destination <c>app-id</c>. The <see cref="HttpRequestMessage.RequestUri" /> 
+        /// property will be replaced with a new URI with the authority section replaced by the instance's <see cref="httpEndpoint"/> value
+        /// and the path portion of the URI rewritten to follow the format of a Dapr service invocation request.
+        /// </para>
+        /// </summary>
+        /// <param name="appId">
+        ///     An optional <c>app-id</c>. If specified, the <c>app-id</c> will be configured as the value of 
+        ///     <see cref="HttpClient.BaseAddress" /> so that relative URIs can be used. It is mandatory to set this parameter if your app-id contains at least one upper letter.
+        ///     If some requests use absolute URL with an app-id which contains at least one upper letter, it will not work, the workaround is to create one HttpClient for each app-id with the app-ip parameter set.
+        /// </param>
+        /// <returns>An <see cref="HttpClient" /> that can be used to perform service invocation requests.</returns>
+        /// <remarks>
+        /// </remarks>
+        #nullable enable
+        public override HttpClient CreateInvokableHttpClient(string? appId = null)
+        {
+            var handler = new InvocationHandler
```

Comment:
Could this just call the existing static method and pass along the endpoint and API key?

Reply:
Yes we could.